src/server/raft_client.rs

```diff
@@ -243,9 +242,9 @@ impl<T: RaftStoreRouter<RocksEngine>> RaftClient<T> {
 struct RaftMsgCollector(usize);
 impl BatchCollector<Vec<RaftMessage>, RaftMessage> for RaftMsgCollector {
     fn collect(&mut self, v: &mut Vec<RaftMessage>, e: RaftMessage) -> Option<RaftMessage> {
-        let mut msg_size = 0;
+        let mut msg_size = e.start_key.len() + e.end_key.len() + mem::size_of::<RaftMessage>();
```

Comment:
Why counting memory size? It has nothing to do with the final result.

Reply:
Most of fields of `RaftMessage` are inlined, so memory size is close to `compute_size`.

Reply:
That's not how compute_size work. For example, zero numbers won't be counted in compute_size.